app/helpers/ui_helper.rb

```diff
@@ -74,9 +74,9 @@ def print_button_to(link, options = {})
   end
 
   # Generic Submit button for a form
-  def submit_button(options = {})
+  def submit_button(options = {}, data = {})
     disable_text = options[:disable_text] || "Saving"
-    _button_to({ text: "Save", icon: "floppy-o", size: "lg", type: "success", align: "pull-right" }.merge(options), data: { disable_text: disable_text })
+    _button_to({ text: "Save", icon: "floppy-o", size: "lg", type: "success", align: "pull-right" }.merge(options), data: { disable_text: disable_text }.merge(data), name: options[:name] || 'button')
```

Comment:
well done -- thanks for doing it this way. :100:

Reply:
:+1: